controllers/downloads.js

```diff
@@ -121,16 +132,22 @@ var getSumOfDays = function(request,reply,conditions,period) {
  * @param period
  */
 var getRangeOfDays = function(request,reply,conditions,period) {
-
   var sql = ''
   var bindValues = []
+  var qmarks = []
+  var bulk = false
 
   bindValues.push(period.start)
   bindValues.push(period.end)
 
   if(conditions.package) {
-    sql += 'SELECT day, downloads FROM downloads WHERE day >= ? and day <= ? AND package = ?'
-    bindValues.push(conditions.package)
+    bulk = conditions.package.indexOf(',') > -1
+    conditions.package.split(',').forEach(function(package, i) {
+      qmarks.push('?')
+      bindValues.push(package)
+    })
+
+    sql += 'SELECT package, day, downloads FROM downloads WHERE day >= ? and day <= ? AND package in (' + qmarks.join(',') + ') GROUP BY package,day'
```

Comment:
You can no longer rely on the days being in order if you are also grouping by package, so you should explicitly order by package and day.